### PR TITLE
fix: market expiry & trading live copy

### LINKED ISSUE #2602: Update market expiry in trade header
## Story

As a trader
I want to know when a market expires
So that I can trade, and close out positions accordingly

## Acceptance Criteria

Given the way the system is designed (where the settlement date is not defined in the market framework) AND users will expect the future market to settle at some point:

For Markets that are futures, when showing a settlement date:

- if the market has already settled, show that
- else if there is a timestamp in the settlement oracle in Vegatime then show that
- else if there is a metadata settlement date tag then show that
- if we've passed the metadata date and the market is still trading show a message that says the market has not received a settled event
- else: show "Settlement date not defined" or something appropriate

### PR DESCRIPTION
**Pull request by MadalinaRaicu**
# Related issues 🔗

Closes #2602 #2612

# Description ℹ️

- [x] update expiry tooltip - if metadata shows expired but the market still not settled/terminated
- [x] update trading live copy 

# Demo 📺
Screenshot 2023-01-16 at 13 04 30

Screenshot 2023-01-16 at 13 32 57

Screenshot 2023-01-16 at 13 33 20

Screenshot 2023-01-16 at 12 20 10

Screenshot 2023-01-16 at 13 48 47

### CODE REVIEW

File: apps/trading/components/welcome-dialog/welcome-landing-dialog.tsx [hunk 1]
```diff
@@ -84,7 +84,9 @@ export const SelectMarketLandingTable = ({
           data-testid="view-market-list-link"
           onClick={() => onClose()}
         >
-          <UILink>{'Or view full market list'} </UILink>
+          <UILink className="uppercase underline">
```
Comment (MadalinaRaicu) [inline apps/trading/components/welcome-dialog/welcome-landing-dialog.tsx:87]: align with design

File: apps/trading/components/welcome-dialog/welcome-notice-dialog.tsx [hunk 2]
```diff
@@ -28,8 +26,7 @@ export const WelcomeNoticeDialog = () => {
       </h1>
       <p className="leading-6 mb-7">
         {t(
-          'Vega %s is now live, but markets need to be voted for before the can be traded on. In the meantime:',
-          [networkName]
+          'Trading on Vega is now live, but markets need to be voted for before they can be traded on. In the meantime:'
```
Comment (MadalinaRaicu) [inline apps/trading/components/welcome-dialog/welcome-notice-dialog.tsx:29]: fix copy

File: libs/market-info/src/components/market-expires/market-expires.tsx [hunk 3]
```diff
@@ -36,25 +37,27 @@ export const getMarketExpiryDateFormatted = (
 };
 
 export const getExpiryDate = (market: SingleMarketFieldsFragment): string => {
-  const closeDate = getMarketExpiryDate(
+  const metadataExpiryDate = getMarketExpiryDate(
```
Comment (MadalinaRaicu) [inline libs/market-info/src/components/market-expires/market-expires.tsx:40]: clearer var names

File: apps/trading/components/liquidity-supplied/liquidity-supplied.tsx [hunk 4]
```diff
@@ -146,5 +143,9 @@ export const MarketLiquiditySupplied = ({
       {supplied} (
       {percentage.gt(100) ? '>100%' : formatNumberPercentage(percentage, 2)})
     </HeaderStat>
+  ) : (
+    <HeaderStat heading={t('Liquidity supplied')} testId="liquidity-supplied">
```
Comment (MadalinaRaicu) [inline apps/trading/components/liquidity-supplied/liquidity-supplied.tsx:147]: Screenshot 2023-01-16 at 13 48 47

File: apps/trading/components/welcome-dialog/welcome-notice-dialog.tsx (hunk 2 above)
Comment (candida-d) [inline apps/trading/components/welcome-dialog/welcome-notice-dialog.tsx:29]:
```suggestion
          'There are no markets to trade on right now. Trading on Vega is now live, but markets need to pass a governance vote before they can be traded on. In the meantime:'
```

File: apps/trading/components/welcome-dialog/welcome-notice-dialog.tsx [hunk 5]
```diff
@@ -28,8 +26,7 @@ export const WelcomeNoticeDialog = () => {
...
         )}
       </p>
       <ul className="list-[square] pl-7">
```
Comment (candida-d) [inline apps/trading/components/welcome-dialog/welcome-notice-dialog.tsx:32]: Below, could you change the link text from 'Propose your own markets' to "Propose a market"?

**Review by sam-keen — APPROVED** [review]

**Pull request merged**